### PR TITLE
Functional tests - Delete unused selectors and methods

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Delete unused selectors and methods
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/*" URL_FO=http://prestashop.local/PrestaShop/  npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19785)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/pages/BO/login/index.js [hunk 1]
```diff
@@ -22,13 +22,13 @@ module.exports = class Login extends BOBasePage {
   /**
    * Enter credentials and submit login form
    * @param email
-   * @param passwd
+   * @param pwd
```
Comment (Progi1984) [inline tests/UI/pages/BO/login/index.js]:
```suggestion
   * @param password
```

File: tests/UI/pages/BO/login/index.js [hunk 2]
```diff
@@ -22,13 +22,13 @@ module.exports = class Login extends BOBasePage {
...
    * @param waitForNavigation, false if login should fail
    * @returns {Promise<void>}
    */
-  async login(email, passwd, waitForNavigation = true) {
+  async login(email, pwd, waitForNavigation = true) {
```
Comment (Progi1984) [inline tests/UI/pages/BO/login/index.js]:
```suggestion
  async login(email, password, waitForNavigation = true) {
```

File: tests/UI/pages/BO/login/index.js [hunk 3]
```diff
@@ -22,13 +22,13 @@ module.exports = class Login extends BOBasePage {
...
     await this.page.type(this.emailInput, email);
-    await this.page.type(this.passwordInput, passwd);
+    await this.page.type(this.passwordInput, pwd);
```
Comment (Progi1984) [inline tests/UI/pages/BO/login/index.js]:
```suggestion
    await this.page.type(this.passwordInput, password);
```

**Review by Progi1984 — COMMENTED** [review]
A small suggestion

File: tests/UI/pages/BO/login/index.js (hunk 3 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/login/index.js]: Done thx :)

File: tests/UI/pages/BO/advancedParameters/team/profiles/index.js [hunk 4]
```diff
@@ -72,7 +72,7 @@ module.exports = class Profiles extends BOBasePage {
 
   /**
    * Reset input filters
-   * @return {Promise<textContent>}
+   * @returns {Promise<integer>}
```
Comment (PierreRambaud) [inline tests/UI/pages/BO/advancedParameters/team/profiles/index.js]: why sometimes int, and sometimes integer? :sweat_smile:

File: tests/UI/pages/BO/catalog/products/index.js [hunk 5]
```diff
@@ -215,7 +214,7 @@ module.exports = class Product extends BOBasePage {
    * Get Text Column
    * @param columnName
    * @param row
-   * @return {Promise<float|string>}
+   * @return {Promise<Float|textContent>}
```
Comment (PierreRambaud) [inline tests/UI/pages/BO/catalog/products/index.js]:
```suggestion
   * @return {Promise<float|textContent>}
```

File: tests/UI/pages/BO/advancedParameters/team/profiles/index.js (hunk 4 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/advancedParameters/team/profiles/index.js]: Thank you :D I will check all returns ;)

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**